HelpMyStreetFE/HelpMyStreetFE/src/sass/components/profile/_job.scss

```diff
@@ -11,100 +11,109 @@
 }
 
 .job {
-  @include shadow-xs;
-  width: 100%;
-  min-height: 50px;
-  border: 1px solid $gray-8;
-  border-radius: 5px;
-  margin: 10px 0;
-  padding: 5px;
-  overflow: hidden;
-
-  @media (min-width: $med-screen) {
-    padding: 10px 5px;
-  }
-
-  &.highlight {
-    animation: highlight 2s;
-  }
-
-  &__info {
-    min-height: 100px;
-    display: grid;
-    grid-template-columns: 1fr;
+    @include shadow-xs;
+    width: 100%;
+    min-height: 50px;
+    border: 1px solid $gray-8;
+    border-radius: 5px;
+    margin: 10px 0;
+    padding: 5px;
+    overflow: hidden;
 
     @media (min-width: $med-screen) {
-      grid-template-columns: 1fr 2fr 1.3fr;
+        padding: 10px 5px;
     }
 
-    div {
-      display: flex;
-      align-items: center;
-      justify-content: center;
+    &.highlight {
+        animation: highlight 2s;
     }
 
-    .column {
-      flex-direction: column;
-    }
+    &__info {
+        min-height: 100px;
+        display: grid;
+        grid-template-columns: 1fr;
 
-    &__urgency {
-      color: $dark-blue;
-      height: 50px;
-      margin: -5px -5px 5px;
-      background-color: $washed-blue;
-
-      span,
-      img {
-        padding: 0 5px;
-      }
-
-      img {
-        height: 25px;
-        margin: 5px;
-      }
-
-      @media (min-width: $med-screen) {
-        height: auto;
-        background-color: transparent;
-        flex-direction: column;
-        justify-content: center;
-      }
-
-      .critical {
-        font-weight: bold;
-      }
-
-      .critical-icon {
-        margin: 5px;
-        display: inline-block;
-        text-align: center;
-        height: 25px;
-        width: 25px;
-        background-color: $orange;
-        border-radius: 25px;
-        color: white;
-      }
-    }
+        @media (min-width: $med-screen) {
+            grid-template-columns: 1fr 2fr 1.3fr;
+        }
 
-    &__activity {
-      .activity__title {
-        font-weight: bold;
-        display: flex;
-        align-items: center;
-        margin: 5px;
-
-        img {
-          display: inline-block;
-          margin-right: 10px;
-          max-width: 32px;
+        div {
+            display: flex;
+            align-items: center;
+            justify-content: center;
         }
-      }
-    }
 
-    &__actions {
-      .btn {
-        width: 170px;
-        padding: 5px 0;
+        .column {
+            flex-direction: column;
+        }
+
+        .flex-end {
```

Comment:
github  showing a lot of changes in SCSS.

all i added was

`    .flex-end {
            margin-top: 0.4rem;

            @media (min-width: $sm-screen) {
                justify-content: flex-end;
                margin-top:0px;
            }
        }
`